proto/eth/v1/validator_service.proto

```diff
@@ -19,14 +19,12 @@ import "google/api/annotations.proto";
 import "google/protobuf/descriptor.proto";
 import "google/protobuf/empty.proto";
 
-import "proto/eth/ext/options.proto";
-import "proto/eth/v1/attestation.proto";
-import "proto/eth/v1/beacon_block.proto";
+import "proto/eth/v1/validator.proto";
 
-option csharp_namespace = "Ethereum.Eth.v1";
+option csharp_namespace = "Ethereum.Eth.V1";
 option go_package = "github.com/prysmaticlabs/prysm/proto/eth/v1";
 option java_multiple_files = true;
-option java_outer_classname = "ValidatorServiceProto";
```

Comment:
is this accurate ? Shouldnt this remain as ValidatorService

Reply:
Thanks for noticing